### DISCUSSION TITLE
Limit `com.atproto.sync.getBlocks` CID array length in lexicon?

### DISCUSSION THREAD

**Discussion by DavidBuchanan314** [bluesky-social/atproto#3269] [Protocol (atproto)]
There's an implicit limit due to the length of the request URL (since they have to go in GET params), but that might vary depending on various factors.

Setting a conservative explicit limit in the lexicon def might improve interop.

**Comment by bnewbold** [comment]
Seems reasonable. Maybe 20 or 25? Probably want to keep things under 2 KByte.

We were also just discussing this endpoint and whether it should be considered part of the "core atproto repo sync" interface (implied by being under `com.atproto.sync.*`), or be considered an extension or "debugging" helper endpoint.
